modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -332,6 +335,7 @@ void EditorSceneFormatImporterBlend::get_import_options(const String &p_path, Li
 	r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::INT, SNAME(PATH), PROPERTY_HINT_ENUM, ENUM_HINT), VALUE));
 
 	ADD_OPTION_ENUM("blender/nodes/visible", "All,Visible Only,Renderable", BLEND_VISIBLE_ALL);
+	ADD_OPTION_BOOL("blender/nodes/active_collection", false);
```

Comment:
This would maybe be clearer? In Blender it's under "Limit To", which makes the meaning clear, but on our end it's not obvious what enabling an option named "Active Collection" would do IMO.
```suggestion
	ADD_OPTION_BOOL("blender/nodes/active_collection_only", false);
```

Reply:
You renamed one occurrence of the option, but there are three (this one, and the `&&` part where it's checked).

Reply:
Oh, I'm sorry for stalling this.